incubator/vault/templates/role.yaml

```diff
@@ -0,0 +1,20 @@
+{{- if .Values.rbac.create -}}
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRoleBinding
+metadata:
+  name: {{ template "vault.serviceAccountName" . }}
+  labels:
+    app: {{ template "vault.name" . }}
+    chart: {{ template "vault.chart" . }}
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+  namespace: default
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: system:auth-delegator
+subjects:
+- kind: ServiceAccount
+  name: {{ template "vault.serviceAccountName" . }}
+  namespace: default
```

Comment:
Shouldn't this use `{{ .Release.Namespace }}`?

Reply:
Good catch! @aravindvalkodaiqlik , would you be able to make a quick follow-up PR to fix this?

A good example of a cluster role binding is here:
https://github.com/helm/charts/blob/master/stable/nginx-ingress/templates/clusterrolebinding.yaml

We also don't need `namespace` as part of the metadata for the cluster role binding. (Line 11)

Lastly, it might make sense to rename the manifest from `role.yml` to `clusterrolebinding.yml`